SDK_dotnet/Models/UnspentOutput.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Gluwa.SDK_dotnet.Models
+{
+    public sealed class UnspentOutput
+    {
+        /// <summary>
+        /// Amount (in Bitcoin) for the transaction output.
+        /// </summary>
+        public string Amount { get; private set; }
+
+        /// <summary>
+        /// Transaction hash of the transaction that contains the output.
+        /// </summary>
+        public string TxHash { get; private set; }
+
+        /// <summary>
+        /// Index of the output in the transaction.
+        /// </summary>
+        public int Index { get; private set; }
+
+        /// <summary>
+        /// Number of confirmations for the transaction.
+        /// </summary>
+        public int Confirmations { get; private set; }
+
+        /// <summary>
+        /// Internal use only.
+        /// </summary>
+        public string Tx { get; set; }
+
+        public UnspentOutput(
```

Comment:
When you test, make sure this deserialize properly. I have a feeling that it won't because the properties are private set and you don't have JsonConstructor attribute on the constructor